client/components/domains/transfer-domain-step/style.scss

```diff
@@ -37,6 +37,10 @@
 	.transfer-domain-step__domain-availability {
 		margin-top: 30px;
 	}
+
+	.domain-suggestion.card {
+		border-left: 3px solid #4ab866;
```

Comment:
`$alert-green` here too.

Reply:
Oops, these should be fixed now. Thank you!